cdap-docs/faq/source/cdap-startup.rst

```diff
@@ -0,0 +1,255 @@
+to installing a parcel:
+
+- **Adding the repository** to the list of repositories searched by Cloudera Manager
+- **Downloading** the parcel to the Cloudera Manager server
+- **Distributing** the parcel to all the servers in the cluster
+- **Activating** the parcel
+
+Start by clicking on the parcel icon (near the top-left corner of Cloudera Manager and looks
+like a gift-wrapped box) and ensure that the CDAP parcel is listed as *Active*.
+
+
+I've followed the install instructions, yet CDAP does not start and fails verification. What next?
+--------------------------------------------------------------------------------------------------
+If you have followed :ref:`the installation instructions <install>`, and CDAP either did not pass the 
+:ref:`verification step <configuration-verification>` or did not startup, check:
+
+- Look in the CDAP logs for error messages (located either in ``/var/log/cdap`` for CDAP Distributed or 
+  ``$CDAP_HOME/logs`` for CDAP SDK Standalone)
+- If you see an error such as::
+
+    ERROR [main:c.c.c.StandaloneMain@268] - Failed to start Standalone CDAP
+    java.lang.NoSuchMethodError: 
+    co.cask.cdap.UserInterfaceService.getServiceName()Ljava/lang/String
+
+  then you have downloaded the standalone version of CDAP, which is not intended
+  to be run on Hadoop clusters. Download the appropriate distributed packages (RPM or
+  Debian version) from http://cask.co/downloads.
+         
+- Check permissions of directories:
+
+  - The :ref:`CDAP HDFS User <configuration-options>` (by default, ``yarn``) owns the HDFS directory (by default,  ``/cdap``).
+  - The :ref:`Kafka Log directory <configuration-options>` (by default, ``/data/cdap/kafka-logs``), must be writable by the default CDAP user.
+  - The :ref:`temp directories <configuration-tmp-files>` utilized by CDAP must be writable by the default CDAP user.
+
+..
+
+- Check YARN using the YARN Resource Manager and see if the CDAP Master services are starting up.
+  Log into the cluster at ``http://<host>:8088/cluster/apps/RUNNING`` and access the YARN Resource Manager webapp. The
+  CDAP Master services should be listed under "RUNNING":
+  
+  .. image:: _images/yarn-rm-running.png
+     :align: center
+     :width: 8in
+  
+..
+
+- If CDAP Master has started, query the backend by using a command (substituting for ``<host>`` as appropriate)::
+
+    $ curl -w'\n' <host>:10000/v3/system/services/status
+    
+  The response should be something similar to::
+  
+    {"dataset.executor":"OK","metrics":"OK","transaction":"OK","appfabric":"OK","metadata.service":"OK",
+     "streams":"OK","explore.service":"OK","log.saver":"OK","metrics.processor":"OK"}
+
+..
+
+- Check that the CDAP UI can reach the CDAP Router (by default, the URL will be
+  ``http://<host>:9999`` where ``<host>`` is the IP address of one of the machines where you
+  installed the packages and started the services).
```

Comment:
Check that the CDAP UI is accessible (by default, the URL will be...

Separately, and probably in the next section (which is really a more specific version of this same question), add:
Ensure that the CDAP UI can connect to the CDAP Router.  Check that the configured ``router.server.address`` and ``router.server.port`` (default 10000) in cdap-site.xml corresponds with where the CDAP router is listening.

(if UI cannot connect to router, then it will show the "namespace cannot be found" msg)